docs/src/main/asciidoc/verifier/introduction.adoc

```diff
@@ -817,3 +817,27 @@ The rest of the flow looks the same.
 
 Yes! Check out the https://cloud.spring.io/spring-cloud-contract/spring-cloud-contract.html#_different_base_classes_for_contracts[Different base classes for contracts] sections
 of either Gradle or Maven plugins.
+
+==== How can I debug the request/response being sent by the generated tests client?
+
+The generated tests all boil down to RestAssured in some form or fashion which relies on https://hc.apache.org/httpcomponents-client-ga/[Apache HttpClient].  HttpClient has a facility called https://hc.apache.org/httpcomponents-client-ga/logging.html#Wire_Logging[wire logging] which logs the entire request and response to HttpClient.  An example log4j2.xml configurtion could look like this
+[source,xml,indent=0]
+----
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Actually I think that it's enough to add in test properties sth like this.

```
logging.level.org.apache.http.wire=DEBUG
```

And then there's no need for any additional logging configuration. WDYT?

Check this out for more info - https://docs.spring.io/spring-boot/docs/current/reference/html/howto-logging.html